src/renderer/processes/multisigs/model/multisigs-model.ts

```diff
@@ -0,0 +1,160 @@
+import { combine, createEffect, createEvent, sample } from 'effector';
+import { interval, once } from 'patronum';
+import { GraphQLClient } from 'graphql-request';
+
+import {
+  Chain,
+  MultisigAccount,
+  NotificationType,
+  AccountType,
+  ChainType,
+  CryptoType,
+  ExternalType,
+  SigningType,
+  WalletType,
+  Wallet,
+  NoID,
+} from '@shared/core';
+import { networkModel, networkUtils } from '@entities/network';
+import { accountUtils, walletModel, walletUtils } from '@entities/wallet';
+import { MultisigResult, multisigService } from '@entities/multisig';
+import { isEthereumAccountId, toAddress } from '@shared/lib/utils';
+import { notificationModel } from '@entities/notification';
+import { multisigUtils } from '../lib/mulitisigs-utils';
+
+type SaveMultisigParams = {
+  wallet: Omit<NoID<Wallet>, 'isActive' | 'accounts'>;
+  accounts: Omit<NoID<MultisigAccount>, 'walletId'>[];
+};
+
+const MULTISIG_DISCOVERY_TIMEOUT = 30000;
+
+const multisigsDiscoveryStarted = createEvent();
+const multisigSaved = createEvent<GetMultisigsResult>();
+
+const $chainsSupportingMultisigDiscovery = combine(networkModel.$chains, (chains) => {
+  return Object.values(chains).filter((chain) => {
+    return multisigUtils.isMultisigSupported(chain) && Boolean(chain.externalApi?.[ExternalType.MULTISIG]?.[0]?.url);
```

Comment:
Seems fine, I would suggest to introduce 2 variables with meaningful names and `return var_1 && var_2`
for readability